install/install.php

```diff
@@ -99,6 +99,31 @@ public function install(Migration $migration, $args = []): bool {
       $task = new CronTask();
       PluginFormcreatorIssue::cronSyncIssues($task);
 
+      // Add rights
+      global $DB;
+      $profiles = $DB->request([
+         'SELECT' => ['id'],
+         'FROM'   => Profile::getTable(),
+      ]);
+      foreach ($profiles AS $profile) {
+         $rights = \ProfileRight::getProfileRights(
+            $profile['id'],
+            [
+               \Entity::$rightname,
+               \PluginFormcreatorForm::$rightname,
+            ]
+         );
+         if ($rights[\Entity::$rightname] >= UPDATE) {
+            \ProfileRight::updateProfileRights($profile['id'], [
+               \PluginFormcreatorForm::$rightname => READ + UPDATE + CREATE + DELETE + PURGE,
+            ]);
+         } else {
+            \ProfileRight::updateProfileRights($profile['id'], [
+               \PluginFormcreatorForm::$rightname => READ,
+            ]);
+         }
+      }
```

Comment:
```suggestion
      foreach ($profiles as $profile) {
         $rights = ProfileRight::getProfileRights(
            $profile['id'],
            [
               Entity::$rightname,
               PluginFormcreatorForm::$rightname,
            ]
         );
         $right = READ;
         if ($rights[Entity::$rightname] & (UPDATE + CREATE + DELETE + PURGE)) {
            $right = READ + UPDATE + CREATE + DELETE + PURGE;
         }
         ProfileRight::updateProfileRights($profile['id'], [
            PluginFormcreatorForm::$rightname => $right,
         ]);
      }
```

Reply:
This suggestion 
- reduces sligthly the code, 
- drops canonical class names (if GLPI core classes move in a namespace this will be risky and painful to update everywhere. It is easier to simply add a **use** statement at the beginning of the file)
- use bitwise masks expressions instead of comparing bit fields as integer values